Makefile

```diff
@@ -132,11 +132,24 @@ godoc: install
 NODEPREFIX = .hermit/node
 NODELIB = $(NODEPREFIX)/lib
 
+define PLAYWRIGHT_CMD
+	npm --prefix e2e ci
+	npx --prefix e2e playwright test --config e2e $(PLAYWRIGHT_ARGS)
+endef
+
+PLAYWRIGHT_IMG = mcr.microsoft.com/playwright:v1.40.0-jammy
+PLAYWRIGHT_CMD_DOCKER = docker run --rm \
+  --volume $$(pwd):/work/ -w /work/ \
+  --network host --add-host=host.docker.internal:host-gateway \
+  --env BASEURL=$(BASEURL) \
+  $(PLAYWRIGHT_IMG) /bin/bash -e -c "$(subst $(nl),;,$(PLAYWRIGHT_CMD))"
```

Comment:
This can stay for now, but seeing it again I realise it's a bi dodgy (like most shell stuff). If `PLAYWRIGHT_CMD` were to contain double quotes, they would foul up this expansion, and double quotes in a shell command is not that uncommon. There likely should be a better way to pass the input - perhaps running docker with `-i` (keep stdin) and have bash read the script from stdin.

But this works for now but perhaps be aware

Reply:
ok, I will try to remember 😅

Reply:
You will remember when  you break it. But at least you should not be going "huh?" when it happens if you recall this conversation.